### PR TITLE
Demo GitHub Flow

### PR DESCRIPTION
**Pull request by githubteacher**
This pull request attempts to demonstrate GitHub Flow, using both the CLI and github.com.

## Title

- [x] overview of concepts
- [x] make sure everyone (or most!) can `git push`

@halfaro1 anything I'm missing?

### CODE REVIEW

**Comment by githubteacher** [comment]
🍣

**Comment by githubteacher** [comment]
If you haven't already, please comment on #3

**Comment by fpugh** [comment]
This is a comment.

File: pbm_bio.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+### Hometown
+- San Francisco
```
Comment (breadraptor) [inline pbm_bio.md]: Nice :sparkles:

**Pull request merged**